resources/views/ama_list.blade.php

```diff
@@ -0,0 +1,97 @@
+@extends('layouts/admin')
+
+
+@section('title')
+
+	AMAs list
+
+@endsection
+
+@section('content')
+		
+	
+              <div class="col-sm-12">
+                <table class="table table-bordered table-striped">
+
+                  <thead>
+                    <tr>
+                      <th>Title:</th>
+                      <th>Answred by:</th>
+                      <th>Tags:</th>
+                      <th>Actions</th>
+                    </tr>
+                  </thead>
+
+                  <tbody>
+                  	@foreach ($amas as $amas)
+                  	<tr>
+                    <td>{{$amas->title}}</td>
+                    <td>{{$amas->person}}</td>
+                    <td>{{$amas->tags}}</td>
+                    <td>
+                    
+                          <button type="button" class="btn btn-info">
+                          	View Details
+                      	  </button>
+                        
+                          <button type="button" class="btn btn-info">
+                          	Edit
+                      	  </button>
+                       
+                          <button type="button" class="btn btn-info">
+                          	Delete
+                          </button>
+                        
+                    </td>
+                	</tr>
+                    @endforeach
+                  </tbody>
+                </table>
+
+
+          <p> Filter data by: </p>
+          	<form action="">
+		        
+		        From:<input type="date" min="2018-01-01" max="2019-12-31" name="date_from"> 
+
+		        To:<input type="date" min="2018-01-01" max="2019-12-31" name="date_to"> 
+		    	</br>
+
+		    		Tags: 
+		    	<select  id="select_tags" name="tags"  style="width: 20%">
+
+  					@foreach($amas as $amas)
+
+  						
+            			<option value=" $amas->tags" > amas </option>
```

Comment:
There are a couple of problems here:
1. You need to use `{{ ... }}` to get the value of the tag, this only prints `$amas->tags`.
2. You get the tags from the amas records and this will return you all tags as they're added to the ama records. That means that if I added `test, ama`, that will be treated as one tag instead of 2. To solve this, create a new variable in the controller for the tags and then use it here in the foreach.